test/test_db.py

```diff
@@ -21,7 +21,7 @@
 #
 
 from ngamsLib import ngamsDb, ngamsDiskInfo, ngamsFileInfo
-from test import ngamsTestLib
+import ngamsTestLib
```

Comment:
This change is breaking the tests in python 3. Was this somehow giving you troubles in Centos 7 with python 2.7?

Reply:
I thought it was completely superfluous. I did not know this was required for python 3. Both test_db.py and ngamsTestLib.py are in the same directory. I am surprised it is needed.
I package the unit test files and place them in a new directory with a different name. Would it be possible to rename the test directory to ngamsTest?

Reply:
> I thought it was completely superfluous. I did not know this was required for python 3. Both test_db.py and ngamsTestLib.py are in the same directory. I am surprised it is needed.

Imports are always absolute in python 3, hence the problem, but they could be relative in python 2, which is why the change works there. Another way of having the import work in both versions is having an explicit *relative* import, like `from . import nagmsTestLib`.

BTW, some tests may work if you run them from within the `test/` directory, but in general they are designed to be run from the directory containing `test/`, as many depend on being able to find things in the `test` package and subpackages.

> Would it be possible to rename the test directory to ngamsTest?

Yes, in principle it's possible to rename the `test` directory to `ngamsTest`, but it would take some effort to make it work. The top-level testing directory *was* actually called `ngamsTest` in the past, but I explicitly renamed it to `test` in an long on-going effort of removing the many `ngams*` prefixes that can be found throughout the code. It should also be possible to make the test code be top-level-directory-name-agnostic by having only relative imports like the one above, plus a few other minor things that would need to be changed.

> I package the unit test files and place them in a new directory with a different name

Would it be acceptable to pack the `test` directory into a directory of your naming? So you would end up with, for example, `ngamsTest/test/test_db.py` in your package. The tests would need to be run from `ngamsTests/`.

Reply:
OK, I will try to package the unit tests into ngamsTest/test and see what happens.
I also dislike the ngams/ngas naming inconsistency. It would be nice to replace ngams with ngas throughout. Hopefully one day we will get there.

Reply:
I have packaged the unit tests into ngamsTest/test. This has now been resolved. I think we can now remove this commit from the pull request. Thanks!